docs/en/observability/observability-ui.asciidoc

```diff
@@ -10,7 +10,7 @@ Whether you’re an analyst or an admin, {kib} makes your data actionable by pro
 three key functions. {kib} is:
 
 * **An open-source analytics and visualization platform.**
-Use {kib} to explore your {es} data, and then build beautiful visualizations and dashboards.
+Use {kib} to explore your {es} data, then build beautiful visualizations and dashboards. 
```

Comment:
Is the formatting on these bullets correct? Typically with bolded things like this, they're followed by a : or - instead of a period.